src/main/moditect/module-info.java

```diff
@@ -1,10 +1,9 @@
-module ezvcard {
+open module ezvcard {
```

Comment:
Must the entire module be declared "open" to resolve the issue with the HTML writer?

Reply:
Yes, as far as I can tell they are using reflection to find and initialize something that has to do with the html - writer. You can't declare a single package as open, the whole module needs to be

Reply:
I think you can: https://stackoverflow.com/a/46482376/13379

Reply:
Fixed in the latest commit. 
opens ezvcard.io.html; looks to do the same thing as you said!